pkg/tenant/profiles.go

```diff
@@ -122,8 +122,21 @@ func (s *Server) ProfileUpdate(c *gin.Context) {
 	// Update all fields that are allowed to be updated
 	req.Normalize()
 	member.Name = req.Name
-	member.ProfessionSegment = req.ProfessionSegment
-	member.DeveloperSegment = req.DeveloperSegment
+
+	// Only error on invalid value, empty string is allowed
+	if member.ProfessionSegment, err = db.ParseProfessionSegment(req.ProfessionSegment); err != nil {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(err))
```

Comment:
Should these be more standardized error responses? What would you recommend for the copy?

Reply:
This is fine for now.